src/Sarif/Core/Run.cs

```diff
@@ -13,6 +13,24 @@ public partial class Run
         private static Invocation EmptyInvocation = new Invocation();
         private static LogicalLocation EmptyLogicalLocation = new LogicalLocation();
 
+        public bool ShouldSerializeColumnKind()
```

Comment:
>ShouldSerializeColumnKind [](start = 20, length = 25)

Wow, I hadn't looked closely at this until now. This is the first time I wondered who was calling these, and I see that JSON.NET has a built-in mechanism for [conditional serialization](https://www.newtonsoft.com/json/help/html/ConditionalProperties.htm).